crates/ruff/src/registry.rs

```diff
@@ -735,6 +739,9 @@ pub enum Linter {
     /// [flake8-annotations](https://pypi.org/project/flake8-annotations/)
     #[prefix = "ANN"]
     Flake8Annotations,
+    /// [flake8-async](https://pypi.org/project/flake8-async/)
+    #[prefix = "ASYNC"]
```

Comment:
We don't have any codes this long, but I don't see a good reason to deviate from upstream in this case 🤷